Don't default Adaptive Pricing on for India-based accounts at Connect OAuth (#5529)
woocommerce/woocommerce-gateway-stripe: changelog.txt

* Update - Improve express checkout load times by preloading the resources
* Tweak - Reduce JS bundle sizes by using native browser features instead of polyfills
* Dev - Memoize the Express Checkout button visibility check within a request

> * Fix - Don't default Adaptive Pricing on when connecting a Stripe account it is unavailable for (e.g. India-based accounts)

Reviewer: I think we can simplify this text, and specify that it only applies to first connections/new accounts.
```suggestion
* Fix - Don't default Adaptive Pricing on when first connecting a Stripe account that is ineligible for Adaptive Pricing
```

Reviewer: Alternatively, we could be explicit about the check for India.
```suggestion
* Fix - Don't default Adaptive Pricing on when first connecting a Stripe account from India
```

Author: The first suggestion is more suitable now that we are checking `is_adaptive_pricing_available_for_account`
